www/install/php/Update-21.04.0-beta.1.php

```diff
@@ -18,3 +18,54 @@
  * For more information : contact@centreon.com
  *
  */
+
+// error specific content
+$versionOfTheUpgrade = 'UPGRADE - 21.04.0-beta.1 : ';
```

Comment:
shouldn't it be a 20.10.x script ?

Reply:
will you create new PRs for the 19.10.x and 20.04.x versions ?

Reply:
Yes of course

Reply:
I removed these versions in the description